### DISCUSSION TITLE
185

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#156] [General]
# 潮流周刊第185期 - 光照之美

封面图拍摄于上周吃完晚饭在办公园区散步随手拍，绿树/玻璃/光照/云/健身结合在一起真好看，特别喜欢这种蓝色，让人心情愉悦。

https://weekly.tw93.fun/posts/185-%E5%85%89%E7%85%A7%E4%B9%8B%E7%BE%8E

**Comment by JabinHao** [comment]
《The World I See》简中刚出就买了, 算是读过的最舒畅的传记了, 很羡慕她的家庭环境, 父母的认知对孩子的成长太重要了

**Reply by tw93** [reply]
是的 我以为也要这么对我的孩子，挺有感触的

**Comment by vychodlc** [comment]
居然是工友
